overrides/default.nix

```diff
@@ -973,11 +973,25 @@ lib.composeManyExtensions [
 
       mypy = super.mypy.overridePythonAttrs (
         old: {
-          MYPY_USE_MYPYC =
-            # is64bit: unfortunately the build would exhaust all possible memory on i686-linux.
-            stdenv.buildPlatform.is64bit
-            # Derivation fails to build since v0.900 if mypyc is enabled.
-            && lib.strings.versionOlder old.version "0.900";
+          patches = (old.patches or [ ]) ++ [
+            # FIXME: Remove patch after upstream has decided the proper solution.
+            #        https://github.com/python/mypy/pull/11143
+            (pkgs.fetchpatch {
```

Comment:
This patch needs a version conditional.

Reply:
Actually now as I looked into it more this patch is likely not needed for compilation itself.

It was added because the fix was done when adding mypy-zope package. This patch though fixes a bug that manifests especially in Nix, where we use large PYTHONPATH instead a single site-packages directory.

Should I remove it then?

Reply:
Yes, please remove it if it's not required.

Reply:
fixed

Reply:
looks like that is necessary; added a version conditional